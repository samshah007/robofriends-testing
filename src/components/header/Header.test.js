import { shallow } from "enzyme";
import React from "react";
import Header from './Header';

it('expect to render Header component', () => {
    expect(shallow(<Header></Header>).length).toEqual(1)
});
it('take snap of Header component', () => {
    expect(shallow(<Header />)).toMatchSnapshot();
});