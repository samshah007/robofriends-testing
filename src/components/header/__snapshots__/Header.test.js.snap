// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`take snap of Header component 1`] = `ShallowWrapper {}`;
